c/floats-cbmc-regression/float_lib1_true-unreach-call.c

```diff
@@ -12,7 +12,7 @@ int main() {
   #else
   // Be careful, result of DBL_MAX+DBL_MAX depens on rounding mode!
   if(!(fpclassify(DBL_MAX+DBL_MAX)==FP_INFINITE)) __VERIFIER_error();
-  if(!(fpclassify(0*(DBL_MAX+DBL_MAX))==FP_NAN)) __VERIFIER_error();
+  if(!(fpclassify(0*(LDBL_MAX+LDBL_MAX))==FP_NAN)) __VERIFIER_error();
```

Comment:
This is a fix for a different problem than the one described in the PR, right?

(I agree with this change, just want to understand what's happening.)

Reply:
It actually uses the same idea: use `long double` constants rather than `double`, and thus the problem magically goes away.